test/tsconfig.json

```diff
@@ -28,13 +28,21 @@
     "skipLibCheck": true,
     "stripInternal": true,
     "resolveJsonModule": true,
-    "noEmit": true
+    "noEmit": true,
+    "paths": {
```

Comment:
😭 I am having a really bad day with TS today 😖 I thought #157 took care of this, but as it turns out `node_modules/multiformats` takes precedence without this